src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1535,6 +1536,11 @@ codet java_bytecode_convert_methodt::convert_instructions(
           symbol.name,
           symbol_table);
 
+        // functions of the String libraries can have code
+        // generated for them
```

Comment:
Rephrase: "The string refinement module may provide a definition for this function"